task/builder/builder.go

```diff
@@ -3,12 +3,11 @@ package builder
 import (
 	"context"
 	"fmt"
+	"github.com/drone/go-task/task/logger"
```

Comment:
Should we use `goimports` as a standard to format `.go` files? I believe it is `goimports` ordering that is being used throughout the repo.

Reply:
done